src/sql/stats.py

```diff
@@ -107,10 +107,21 @@ def _summary_stats_parallel(conn, table, column, with_=None):
     try:
         values = conn.execute(query, with_).fetchone()
     except ProgrammingError as e:
+        msg_numeric_type = "You might need to add explicit type casts"
         print(e)
-        raise exceptions.RuntimeError(
-            f"\nEnsure that percentile_disc function is available on {driver}."
-        )
+
+        if (
+            type(e.orig).__name__ == "UndefinedFunction"
+            or type(e.orig).__name__ == "BinderException"
+        ) and msg_numeric_type in e.orig.args[0]:
```

Comment:
comment on when each one is raised, I'm guessing one is for postgres and another one for duckdb?